build/install_as_php_client_linux.sh

```diff
@@ -0,0 +1,175 @@
+#!/bin/bash -m
+
+set +e
```

Comment:
Should -e and -x be used instead?

https://www.gnu.org/software/bash/manual/html_node/The-Set-Builtin.html#:~:text=Using%20%E2%80%98%2B%E2%80%99%20rather%20than%20%E2%80%98%2D%E2%80%99%20causes%20these%20options%20to%20be%20turned%20off.%20The%20options%20can%20also%20be%20used%20upon%20invocation%20of%20the%20shell.%20The%20current%20set%20of%20options%20may%20be%20found%20in%20%24%2D.